tests/ctst/features/utilization/UtilizationAPIs.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Scality Utilization Reporting (SUR) API
+  The utilization metrics API allows different authorized users to retrieve 
+  metrics for buckets, accounts, and locations.
+
+    @2.11.0
+    @PreMerge
+    @Utilization
+    @UtilizationAPI
+    Scenario Outline: Default roles can retrieve utilization metrics
+        Given a STORAGE_MANAGER type
```

Comment:
(not really related to this PR)

this "given" is not very clear : should it not read something like "Given a user with STORAGE_MANAGER role" ?
(probably worth a dedicated PR though, to avoid mixing concerns)

Reply:
Yes we can rename the step indeed... Not sure this is the highest priority though.
I created https://scality.atlassian.net/browse/ZENKO-4999 (next one will be 5000 :eyes: )